i18n/fr/messages.fr.xlf

```diff
@@ -1147,8 +1147,8 @@
         </context-group>
       </trans-unit>
       <trans-unit id="14a5136cbdeef6ed97abfd58112f979f20f619f1" datatype="html">
-        <source><x id="INTERPOLATION" equiv-text="{{ isPinned() ? &quot;Unpin&quot; : &quot;Pin&quot; }}"/></source>
-        <target state="new"><x id="INTERPOLATION" equiv-text="{{ isPinned() ? &quot;Unpin&quot; : &quot;Pin&quot; }}"/></target>
+        <source><x id="INTERPOLATION" equiv-text="{{ isPinned() ? &quot;Détacher&quot; : &quot;Épingler&quot; }}"/></source>
```

Comment:
`<source>` should not be translated. 😉

Reply:
Japanese translation team also faces the problem that this text is not translated in dashboard. We registered issue as #4303 .